packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -417,11 +417,6 @@ export default class TradeStore extends BaseStore {
                 } else if (response.error) {
                     // using javascript to disable purchase-buttons manually to compensate for mobx lag
                     this.disablePurchaseButtons();
-                    this.root_store.common.services_error = {
-                        type: response.msg_type,
-                        ...response.error,
-                    };
-                    this.root_store.ui.toggleServicesErrorModal(true);
```

Comment:
@aliafsah-fs why are we deleting this? we still want to handle other errors that come on purchase, please return this

Reply:
you're right I will make invalid token an exception on this.